kani-compiler/src/codegen_cprover_gotoc/codegen/statement.rs

```diff
@@ -440,7 +440,13 @@ impl<'tcx> GotocCtx<'tcx> {
                     | InstanceDef::ReifyShim(..)
                     | InstanceDef::ClosureOnceShim { .. }
                     | InstanceDef::CloneShim(..) => {
-                        let func_exp = self.codegen_operand(func);
+                        // We need to codegen the operand normally even though we discard the result
+                        // because we need to ensure the symbol table is updated correctly.
+                        // (When deleted, the function `lang_start_internal` is not found anymore when running `cargo kani`.)
+                        let _ = self.codegen_operand(func);
```

Comment:
Is this the same issue as above

Reply:
I was actually able to remove this code now. I'm not sure why it wasn't working before.